weave-js/src/common/components/TraceCountDisplay.tsx

```diff
@@ -17,6 +17,7 @@ export const TraceCountDisplay = ({
     project,
     {},
     undefined,
+    undefined,
```

Comment:
Even thought it is more verbose, I think it could be nicer to do something like `const limit = undefined;` or just add an in-line comment so future devs won't have to resort to hovering to see what these parameters mean.